tests/KafkaFlow.IntegrationTests/Core/Handlers/MessageStorage.cs

```diff
@@ -118,6 +125,22 @@ public static async Task AssertMessageAsync(byte[] message)
             await Task.Delay(100).ConfigureAwait(false);
         }
     }
+    
+    public static async Task AssertNullMessageAsync()
```

Comment:
Here, this should instead validate if nullMessages are an empty byte[] instead of null string

Instead, you can consider the following:

```csharp
public static async Task AssertNullMessageAsync()
{
    var start = DateTime.Now;
    while (!s_nullMessages.IsEmpty)
    {
        if (DateTime.Now.Subtract(start).Seconds > TimeoutSec)
        {
            Assert.Fail("Null message not received");
            return;
        }

        await Task.Delay(100).ConfigureAwait(false);
    }
}
```